README.md

```diff
@@ -1,48 +1,16 @@
 <a name="readme-top"></a>
```

Comment:
_note:  Since there is no open PR in the [frontend](https://github.com/businesskaniki/experts_hub) for me to review, I'll include my comments for that project here.  Basing my review on the  `dev` branch._

# Frontend review comments

- [ ]  On the "License" section of your FRONTEND readme, the link is broken because it points to a file that does not exist in your project.

## Tips and extra help

Go [here](https://github.com/microverseinc/readme-template), grab the `MIT.md` file from that repo, paste it on the root of your project, and link to it in the "License" section of your readme.

That way, your app will be well documented 💪

- [ ] Your frontend app includes no tests, but there is documentation on your readme about how to run them:

![image](https://user-images.githubusercontent.com/57421823/217853510-db73dfd4-01a1-4c25-a3e7-a4f29997512a.png)

Kindly, either add some tests or remove this section to prevent confusion. 👍 

- [ ]  Please, kindly refactor this `App` component to ES6 arrow function, so your code complies with the latest standards.

Refactoring is easy:

```javascript
const App = () => {
  // ...
}
```

## Explanation

The `function` keyword has been soft deprecated in almost all the software industry in favor of ES6 arrow functions. This means, almost nobody uses it anymore, so much that it's considered non-best practice.

That way, your code will meet latest standards. 💪 

- [ ] This script does not exist on your `package.json`: 

![image](https://user-images.githubusercontent.com/57421823/217855541-a6b5c958-2adc-424e-8f96-ac8f9a9d45ff.png)

The correct command is `npm start`.  Kindly correct this on your readme file.

- [ ] When registering to the app, I can see it worked, but only because I had dev tools open:

![image](https://user-images.githubusercontent.com/57421823/217856846-45695318-1c83-4be3-8de5-493f55c1eb32.png)

Aside from that, there is no way to know that a user registration worked.  Kindly provide some way to know if the user was actually registered.  One way to remedy this is to redirect to the `login` page after a successful registration.

That way, your UX will be vastly improved. 👍 

- [ ] While I'm logged in, if I try to make a reservation the frontend tells me it worked, but dev-tools says otherwise:

![image](https://user-images.githubusercontent.com/57421823/217858726-fce54c19-c886-4157-99eb-bf5b40086921.png)

Also, "my reservations" page remains empty:

![image](https://user-images.githubusercontent.com/57421823/217859088-5af987fc-74f3-454b-b7ea-5104958a0d42.png)

(Tested both on Firefox and Chrome).

The server thinks I'm not logged in.  If this is a bug, kindly fix it so your app's reservations work as intended. 👍 

- [ ] When I'm logged in, the app says my name is "John Doe":

![image](https://user-images.githubusercontent.com/57421823/217860228-40851226-88d5-4eff-8a3d-97dd3654b5ac.png)

Kindly fix this in order to improve the UX. 👍 

- [ ] In the mobile view, there is no way to access the "My reservations" or "Add technician" pages.  Kindly fix this so the app is totally responsive.  If there is an opening nav bar, kindly make it more obvious so your users can find it to navigate easily. 👍 

- [ ] You included a search box, however, it does not seem to work:

![image](https://user-images.githubusercontent.com/57421823/217862314-5f85ab9e-2e07-49ae-b61a-1c82fd247487.png)

Make sure it works so your UX is improved.  Tip: You can use a controlled form component. 💪 

- [ ] On your footer:

![image](https://user-images.githubusercontent.com/57421823/217862910-fbb9d37c-91c5-4c5f-a0db-567e801104e7.png)

Kindly implemented some info on your footer.

You could add the name of your team, your team members, or whatever you wish.  Just make sure it includes something meaningful.  

That way, your frontend app will look beautiful. 👍